pepper-apis/src/main/java/org/broadinstitute/ddp/db/dao/UserGovernanceCachedDao.java

```diff
@@ -0,0 +1,79 @@
+package org.broadinstitute.ddp.db.dao;
+
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+import org.broadinstitute.ddp.cache.ModelChangeType;
+import org.broadinstitute.ddp.model.governance.Governance;
+import org.jdbi.v3.core.Handle;
+
+public class UserGovernanceCachedDao extends SQLObjectWrapper<UserGovernanceDao> implements UserGovernanceDao {
+    public UserGovernanceCachedDao(Handle handle) {
+        super(handle, UserGovernanceDao.class);
+    }
+
+    public Governance createGovernedUser(long clientId, long proxyUserId, String alias) {
+        var result = target.createGovernedUser(clientId, proxyUserId, alias);
+        notifyModelUpdated(ModelChangeType.USER, proxyUserId);
+        return result;
+    }
+
+    public Governance createGovernedUserWithGuidAlias(long clientId, long proxyUserId) {
+        var result = target.createGovernedUserWithGuidAlias(clientId, proxyUserId);
+        notifyModelUpdated(ModelChangeType.USER, proxyUserId);
+        return result;
+    }
+
```

Comment:
Here we have another wrapped/cached Dao, but we are actually not even trying to cache data.
Issue here is that this DAO changes User data that might or might not affect be cached somewhere.
Since Jdbi does not maintain for us a relationship between tables and entities like some ORMs would, we need to manually figure out what could.
Turns out that in the Token filter we actually look up user governances so if governances get modified, we don't want that cache data to keep being used. So there is that notifyModel thing.
I think this can be done nicer using a D.I. framework with annotations, but this what it would have to do.